frontend/src/static/js/v_summary.js

```diff
@@ -650,12 +650,28 @@ window.vSummary = {
       user.dailyCommits.forEach((commit) => {
         const { date } = commit;
         if (date >= sinceDate && date <= untilDate) {
-          user.commits.push(commit);
+          const filteredCommit = JSON.parse(JSON.stringify(commit));
```

Comment:
This is to ensure that the filtering of `commitResults` within `commit` does not affect the original `commit`.